library/src/main/java/net/yslibrary/licenseadapter/internal/LicenseViewHolder.java

```diff
@@ -34,6 +34,7 @@ public void bind(@NonNull ExpandableLibrary library) {
     boolean expanded = library.isExpanded();
 
     licenseName.setText(expandableLibrary.getLibrary().getLicense().getName());
+    license.setText(R.string.license_loading);
```

Comment:
Previously when a LicenseViewHolder is reused, license TextView is not updated until license content is loaded. That's why a case like #58 happend.